### DISCUSSION TITLE
Hoop exec command removed

### DISCUSSION THREAD

**Discussion by aaronrenner** [hoophq/hoop#541] [Q&A] [answered]
I noticed in this [recent PR](https://github.com/hoophq/hoop/pull/512), the `hoop exec` command was removed. I'm trying to get just-in-time approval working for a one-off script so I can approve exactly the command the user wants to run without giving them interactive access.

The hoop exec command [here](https://hoop.dev/docs/quickstarts/elixir-IEx#how-to-use) seems to be exactly what I need, but since that command no longer exists, what would I use as the replacement?

**Comment by sandromello** [comment, marked as answer]
You could use the Webapp to perform one-off executions, you just need to visit your public gateway address (`$API_URL/client`), select your connection and execute it.

**Reply by aaronrenner** [reply]
Ah, I didn't realize that. Thank you.
